frontend/src/pages/projects/screens/detail/notebooks/getNotebookImageVersionData.ts

```diff
@@ -0,0 +1,132 @@
+import { ImageStreamKind, ImageStreamSpecTagType, NotebookKind } from '~/k8sTypes';
+import { PodContainer } from '~/types';
+
+export const getCurrentAndLatestNotebookImageVersionData = (
```

Comment:
this duplicates a lot of functions already done in getNotebookImageData.  we should look to reuse that here